docs/pieces/create_pieces.md

```diff
@@ -371,7 +371,7 @@ The dependency files are stored in a separate folder inside the [Pieces reposito
 Optionally, you can also include in the metadata:
 
 - style configurations for the visual node representing `MyNewPiece` in the Domino GUI, such as label and icon. Currently the available icons are the free options from [Font Awesome v5](https://fontawesome.com/v5/search?m=free).
-- minimal and limit resources required to run the Piece, when running Domino in Kubernetes
+- minimal and limit resources required to run the Piece, when running Domino in Kubernetes. The units are millicores (m) for CPU and milibyte (Mi) for memory.
```

Comment:
isn't this megabytes for memory? @vinicvaz

Reply:
No, it is milicore for cpu and mebibyte for memory.
Refs: https://medium.com/swlh/understanding-kubernetes-resource-cpu-and-memory-units-30284b3cc866
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes